q2_composition/tests/test_ancombc2.py

```diff
@@ -137,6 +138,49 @@ def test_spaces_in_metadata_column(self):
         # three levels, so one reference and other two in the output
         self.assertEqual(len(variable_with_spaces_columns), 2)
 
+    def test_ancombc2_visualizer(self):
+        '''
+        Tests that the visualizer runs successfully, which is essentially a
+        test of whether or not the visualizer got built into the
+        q2_composition/_ancombc2_visualizer/dist/ folder successfully.
+        '''
+        abc2_output = ancombc2(
+            table=self.biom_table,
+            metadata=self.metadata,
+            fixed_effects_formula='body-site + year',
+            group='body-site',
+            structural_zeros=True
+        )
+
+        with tempfile.TemporaryDirectory() as tempdir:
+            ancombc2_visualizer(tempdir, abc2_output)
```

Comment:
```suggestion
            ancombc2_visualizer(tempdir, abc2_output)
            assert os.path.exists(os.path.join(tempdir, 'index.html'))
```

or something like that